.env.local.example

```
@@ -6,14 +6,14 @@ DIRECT_URL=postgresql://user:password@localhost:5432/dbname
-# Seed
+# 初期管理者アカウント
 SEED_ADMIN_EMAIL=admin@example.com
 SEED_ADMIN_NAME=Administrator
 SEED_ADMIN_PASSWORD=P@ssw0rd
+
+# ユーザーインポート時のデフォルトパスワード
+DEFAULT_USER_PASSWORD=P@ssw0rd
+
+# ゲストのログイン情報
+GUEST_USER_EMAIL=guest@example.com
+GUEST_USER_PASSWORD=P@ssw0rd
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)